lane.py

```diff
@@ -1908,9 +1909,14 @@ def customlist_filter_clauses(
 
 @event.listens_for(Lane, 'after_insert')
 @event.listens_for(Lane, 'after_delete')
-@event.listens_for(Lane, 'after_update')
 @event.listens_for(LaneGenre, 'after_insert')
 @event.listens_for(LaneGenre, 'after_delete')
-@event.listens_for(LaneGenre, 'after_update')
 def configuration_relevant_lifecycle_event(mapper, connection, target):
     site_configuration_has_changed(target)
+
+
+@event.listens_for(Lane, 'after_update')
+@event.listens_for(LaneGenre, 'after_update')
+def configuration_relevant_update(mapper, connection, target):
+    if directly_modified(target):
+        site_configuration_has_changed(target)
```

Comment:
This is the bug fix. An "after_update" doesn't necessarily mean anything has actually been updated in database terms. We ran into this earlier with our other data model objects, which is why we created `directly_modified`.